src/brain/issueLabelHandler/followups.ts

```diff
@@ -83,28 +101,41 @@ export async function ensureOneWaitingForLabel({
     name: 'issueLabelHandler.ensureOneWaitingForLabel',
   });
 
-  const reasonsToDoNothing = [ isFromABot, isNotInARepoWeCareAboutForFollowups ];
+  const reasonsToDoNothing = [ isNotInARepoWeCareAboutForFollowups, isWaitingForLabel ];
```

Comment:
We want to ensure that if getsantry bot applies `Waiting for: *` labels, there aren't duplicates too. We can effectively also skip this step if the label isn't a `Waiting for: *` label